Extension/i18n/chs/package.i18n.json

```diff
@@ -29,9 +29,10 @@
 	"c_cpp.command.RunCodeAnalysisOnActiveFile.title": "在活动文件上运行 Code Analysis",
 	"c_cpp.command.RunCodeAnalysisOnOpenFiles.title": "在打开的文件上运行 Code Analysis",
 	"c_cpp.command.RunCodeAnalysisOnAllFiles.title": "在所有文件上运行 Code Analysis",
```

Comment:
@Colengms Is there a bug on "Code Analysis" not being translated (it seems inconsistent with RemoveAlloCodeAnalysisProblems).